htdocs/product/class/api_products.class.php

```diff
@@ -1704,6 +1841,14 @@ private function _fetch($id, $ref = '', $ref_ext = '', $barcode = '', $includest
             $this->product->sousprods = $childs;
         }
 
+		if ($includeparentid) {
+			$prodcomb = new ProductCombination($this->db);
+			$this->product->fk_product_parent = null;
+			if (($fk_product_parent = $prodcomb->getFkProductParentByFkProductChild($this->product->id)) > 0) {
```

Comment:
This method seems to not exists.
However, the method fetchByFkProductChild() seems to be available to return the expected ID.

Reply:
@eldy we need to resolve this before #14631 https://github.com/Dolibarr/dolibarr/pull/14631#issuecomment-691510241